pymove/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 'e') (D415)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5539342)